dino_catalog/dinocontroller.rb

```diff
@@ -0,0 +1,80 @@
+def controller_home(input)
+  case input
+    when "view"
+      view_show_dinos(@dinodex.dinosaurs)
+    when "save"
+      view_save_files
+    when "add"
+      view_add_dino
+    when "search"
+      view_search_dinos
+    when "exit"
+      exit
+    else
+      begin
+        raise "That's not a valid option!"
+      rescue
+      end
+      view_home
```

Comment:
AWESOME! I don't know what that means!  I'll add it to the list of things to read about!  :)